README.md

```diff
@@ -1,192 +1,154 @@
+### Directory Structure
 
-To view the currently featured charts
-```bash
-bin/partner-charts-ci feature list
-```
-To feature a chart
-```bash
-bin/partner-charts-ci feature add suse/kubewarden-controller 2
-```
-To remove the featured annotation
-```bash
-bin/partner-charts-ci feature remove suse/kubewarden-controller
 ```
-To hide the chart
-```bash
-bin/partner-charts-ci hide suse/kubewarden-controller
+configuration.yaml                      configuration for partner-charts-ci
+index.yaml                              the index.yaml for the helm repository
+packages/                               contains package directories
+  suse/
+    kubewarden-controller/              referred to as a "package directory"
+      upstream.yaml                     configuration specific to a single package
+      overlay/
+        app-readme.md
+        questions.yaml
+    ...
+  ...
+charts/                                 unarchived chart versions
+  suse/
+    kubewarden-controller/
+      1.2.3/
+        ...
+      1.2.4/
+        ...
+      ...
+    ...
+  ...
+assets/                                 archived chart versions
+  suse/
+    kubewarden-controller-1.2.3.tgz
+    kubewarden-controller-1.2.4.tgz
+    ...
+  ...
 ```
 
-After any of those changes, simply add, commit, push - and open a PR with "[modified charts]" in the title
-```bash
-git add index.yaml assets charts
-git commit -m "Hiding suse/kubewarden-controller"
-git push origin main-source
 
-# Open a Pull Request
-```
+### Package Directories
+
+Each package has a package directory. Package directories are of the form
+`packages/<vendor>/<chart>/`, where `<vendor>` is the vendor that provides the chart
+and `<chart>` is the name of the chart. Taken together, `<vendor>/<chart>` is the full
+name used to refer to the package.
 
+Package directories must contain an [`upstream.yaml`](#upstreamyaml) file. Typically
+they also contain an `overlay/` directory that contains files that are copied into
+the chart directory when integrating new versions of the chart. `overlay/` is most
+often used for adding [`app-readme.md`](#app-readmemd) and
+[`questions.yaml`](#questionsyaml) files.
 
 
-## Chart Submission Process
-1. Fork the [Rancher Partner Charts](https://github.com/rancher/partner-charts/) repository
-2. Clone your fork
-3. Ensure the 'main-source' branch is checked out
-4. Create subdirectories in **packages** in the form of *vendor/chart*
-5. Create your **upstream.yaml**
-6. Create any add-on files like your app-readme.md and questions.yaml in an 'overlay' subdirectory (Optional)
-6. Commit your packages directory
-7. Push your commit and open a pull request
+## Example Workflow for Adding a Package
+
+These steps assume that the working directory is set to the root of
+the repository you want to operate on. They use the fictional
```

Comment:
Not exactly _fictional_, just not something that's in this repo. Suggest replacing "fictional" with "example"